runtime/amplitude/src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub mod xcm_config;
 pub mod zenlink;
 
 use crate::zenlink::*;
-use xcm::v1::MultiLocation;
```

Comment:
There is a[ forceful change of `v1::MultiLocation` to `v3::MultiLocation`](https://github.com/paritytech/polkadot/pull/4097) , which happened in a version 0.9.38. 

As stated in [Polkadot's release analysis:](https://forum.polkadot.network/t/polkadot-release-analysis-v0-9-38/2122)
> **How does this impact the Polkadot builders?**
Very unlikely, but in case your parachain runtime is still using v0 or v1 versions, you must upgrade at least to v2 since v0 and v1 will no longer be supported.
This PR as well as its companion introduces `transaction_version` breaking changes, meaning that applying these changes is enough for `transaction_version` to be _bumped_.

* Based on the [0.9.38 release notes](https://github.com/paritytech/polkadot/releases/tag/v0.9.38), there's no migration needed. 
* version [0.9.39](https://github.com/paritytech/polkadot/releases/tag/v0.9.39-rc6) has updates only at runtime
* version [0.9.40 ](https://github.com/paritytech/polkadot/releases/tag/v0.9.40) has runtime migration, but only for messagequeue, nft and nomination pool.

Reply:
Okay, thanks!